x-pack/filebeat/module/kubernetes/coredns/ingest/pipeline.json

```diff
@@ -0,0 +1,18 @@
+{
+  "description": "Pipeline for normalizing Kubernetes coredns logs",
+  "processors": [
+    {
+      "script": {
+        "lang": "painless",
+        "source": "ctx.event.created = ctx['@timestamp']; ctx['@timestamp'] = ctx['coredns']['timestamp']; ctx.coredns.remove('timestamp');",
+        "ignore_failure" : true
```

Comment:
You should parse the date in ingest node, to ensure the document content is consistent. Even if the Elasticsearch `date` datatype is able to parse many things, we want the source document itself to be consistent.

It's a thing ECS will have to make more clear in the future, IMO. (See what happened with the Zeek Unix ts)